cc_plugin_ugrid/resources/fvcom_reference.cdl

```diff
@@ -0,0 +1,110 @@
+netcdf fvcom_reference {
```

Comment:
change to match the associated .nc file:

`netcdf fvcom {`

and update .cdl filename, better to have a clear map between what is responsible for .nc

Reply:
@brianmckenna So which name do you want me to keep?